tracer/test/Datadog.Trace.OpenTracing.Tests/PublicApiTests.cs

```diff
@@ -0,0 +1,126 @@
+// <copyright file="PublicApiTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using FluentAssertions;
+using PublicApiGenerator;
+using Xunit;
+
+namespace Datadog.Trace.OpenTracing.Tests
+{
+    public class PublicApiTests
```

Comment:
[Nit] Maybe consider factoring with the other test (not really convinced it worth it but 🤷 )

Reply:
I think it probably makes sense staying here, as it's about the `Datadog.Trace.OpenTracing` package.

Reply:
Basically what I suggested was to create this class as an abstract class in testhelper, and just override the assemblyname and the snapshot files in the descendants (IIRC that works). But It's only for tests so clearly not a big deal.

Reply:
Pierre, I implemented your suggestion in 2a6473380ae44d93170168f510f79e1214e5c2ab